enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -3173,7 +3198,9 @@ func (p *PebbleCache) reader(ctx context.Context, db pebble.IPebbleDB, r *rspb.R
 	unlockFn := p.locker.RLock(key.LockID())
 	iter := db.NewIter(nil /*default iterOptions*/)
 	defer iter.Close()
-	fileMetadata, err := p.lookupFileMetadata(ctx, iter, key)
+	fileMetadata := rfpb.FileMetadataFromVTPool()
```

Comment:
There's some code in `reader` that hangs on to some pieces of metadata after the function returns. Is that safe in the context of proto reuse? The two cases I saw were:
 - `newChunkedReader` that accesses the chunks in background goroutines
 - `NewDecrypter` that keeps a reference to the digest from the metadata

Reply:
You made a good point. I changed back to use &rfpb.FileMetadata{} in the reader to be less error prone.